internal/service/network/bridge_driver_test.go

```diff
@@ -0,0 +1,123 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
as it is touching system files i think this might require e2e test, especially for failure and rollback cases.

Reply:
I will add this in a follow up PR